src/Doc/ClassDoc.js

```diff
@@ -17,6 +18,32 @@ export default class ClassDoc extends AbstractDoc {
     this['@interface']();
     this['@extends']();
     this['@implements']();
+    this['@member']();
+  }
+
+  ['@member']() {
+    let values = this._findAllTagValues(['@member']);
+    if (!values) return;
+
+    this._value.members = [];
+    for (let value of values) {
+      let {typeText, paramName, paramDesc} = ParamParser.parseParamValue(value);
+      let result = ParamParser.parseParam(typeText, paramName, paramDesc);
+      let types = result.types;
+      this._addExtraValue({
+        __docId__: ASTNodeContainer.addNode(this._node),
```

Comment:
I'm not sure if this is the right way to do it. It would be better to reuse the MemberDoc class, but it expects an ast node which we obviously don't have here.
What is __docId __ used for anyway ?